### DISCUSSION TITLE
Metadata Action

### DISCUSSION THREAD

**Discussion by RichiCoder1** [renovatebot/renovate#24833] [Suggest an Idea]
### What would you like Renovate to be able to do?

Dependabot provides a very helpful [metadata action](https://github.com/dependabot/fetch-metadata) for getting metadata about the current PR. 

This is helpful for when you want to scope down things like approvals, run post scripts for specific packages, or other such cases.

I wondered if there was a way to accomplish this today and if renovate might vendor (or accept the contribution of) an action like this.

### If you have any ideas on how this should be implemented, please tell us here.

The dependabot action does this by parsing well formed metadata included in the commit message. In theory, this could be implemented in renovate using the same method by outputting change details in a machine readable format to either (or both) the commit message and/or the Pull Request details.

### Is this a feature you are interested in implementing yourself?

Maybe

**Comment by RichiCoder1** [comment]
Related/Enabled By: https://github.com/renovatebot/renovate/issues/3727

**Comment by lmilbaum** [comment]
Found an related GitHub discussion -  community/community#27939
